make_prg_from_msa.py

```diff
@@ -598,48 +500,57 @@ def main():
     parser.add_argument("-f", "--alignment_format", dest='alignment_format', action='store', default="fasta",
                         help='alignment_Format of MSA, must be a biopython AlignIO input alignment_format. See '
                              'http://biopython.org/wiki/AlignIO. Default: fasta')
-    parser.add_argument("--max_nesting", dest='max_nesting', action='store', type=int, default=5,
-                        help='Maximum number of levels to use for nesting. Default: 5')
+    parser.add_argument("--max_nesting", dest='max_nesting', action='store', type=int, default=10,
```

Comment:
In the parameter sweep I did for my simulations, we found 5 performed best.

Reply:
Done